ynr/apps/official_documents/models.py

```diff
@@ -40,6 +42,10 @@ class OfficialDocument(TimeStampedModel):
         default="",
     )
 
+    def save(self, *args, **kwargs):
+        self.uploaded_file = convert_sopn_to_pdf(self.uploaded_file)
```

Comment:
@symroe I have captured the pandoc exception in the form validation, so that we can display an error to the user. But wasn't sure if we also want to capture it here (e.g. when running management command to import from the spreadsheet) or if we want it to fail so that we have to go and investigate?

Reply:
Yeah, we really don't want a single failing file to break the import for everything else. Either add here, or catch the exception in the management command?

https://github.com/DemocracyClub/yournextrepresentative/blob/master/ynr/apps/candidates/management/commands/candidates_import_statements_of_persons_nominated.py#L163-L168